src/source/examples/put-campaign-cpc-v1-example.rst

```diff
@@ -0,0 +1,6 @@
+.. code-block:: javascript
+
+    PUT /api/sellside/campaign/123/cpc/10
```

Comment:
wait, do we even support campaignCPC atm or is this just a shortcut to set it on all the linkedAds?

Reply:
not exposed :)

Reply:
and indeed we stopped supporting